docker/xud/xud.conf

```diff
@@ -0,0 +1,24 @@
+lndDir = "~/.lnd"
+rpcPort = 8886
+
+[db]
+username = "xud"
+password = ""
+database = "xud"
+port = 3306
+host = "db"
+
+[p2p]
+listen = true
+port = 8885
+
+[lnd]
+disable = false
+port = 10009
+hostname = "localhost"
+rpcprotopath = "lndrpc.proto"
+
+[raiden]
+disable = false
+port = 5001
+hostname = "localhost"
```

Comment:
New line (if we keep this file).

Reply:
it's required when we are running in a Docker container for connectivity between MySQL and XUD containers. we actually mount this file into the container volume for xud to pickup `mysql`, `lnd` & `raiden` configurations. I'll add a new line.

Reply:
Perhaps just add the required config override, and leave the default values out.

Reply:
Agree with @moshababo, that's been my thinking all along and I believe that lines up with what we discussed, right? Keep the overridden config properties, certainly. But drop the lines that are repeating the default values. The main reason for doing this is so that, if our default values change in the future, we don't have to update this file as well.